docs/learn/glossary.md

```diff
@@ -0,0 +1,114 @@
+
+### Nodes
+Nodes are software libraries that provide core protocol functionality for the Ceramic [network](#networks). Nodes are responsible for processing [stream](#streams) updates (in the form of [signed commits](#signed-commit) from [clients](#clients)), storing [state](#state) for the streams that it cares about, responding to queries, networking with other nodes, replicating streams across the network, and sending valid signed commits to an external [anchor service](#anchor-service) for generating [anchor commits](#anchor-commit).
+
+### Anchor service
+Anchor services are hosted "layer-2" services for Ceramic that generate [anchor commits](#anchor-commit) for many different [streams](#streams) in a scalable, low cost manner by batching many different stream transactions into a merkle tree, and including the merkle root into a transaction on a blockchain platform (currently [Ethereum](#ethereum)). This eliminates the need for each stream transaction to have its own corresponding blockchain transaction, which would be slower and more expensive.
+
+### Networks
+Networks are collections of Ceramic [nodes](#nodes) that share specific configurations and communicate over a dedicated [libp2p](#libp2p) topic. Networks are discrete from one another. [Streams](#streams) that exist on one network are not discoverable on or portable to another. Currently, Ceramic has three primary networks: [mainnet](#mainnet), [Clay Testnet](#clay-testnet), and [dev unstable](#dev-unstable).
+
+### Mainnet
+Mainnet is the Ceramic [network](#networks) used for production deployments. For more information on mainnet, see the Networks page.
+
+### Clay Testnet
+Clay Testnet is a Ceramic [network](#networks) used for community testing and development purposes. For more information on Clay Testnet, see the Networks page.
+
+### Dev Unstable
+Dev Unstable is a Ceramic [network](#networks) used by Ceramic core protocol developers for testing new protocol features and release candidates. It should be considered unstable and highly experimental.
+
+
+## Underlying technologies
+
+### IPFS
+[IPFS](https://ipfs.io) is the Interplanetary File System. Simply put, IPFS is a way to address static content using [CIDs](#cid) and to discover this content over a peer-to-peer network of nodes. Ceramic relies on IPFS for storing the [commits](#commits) that make up [streams](#streams) and discovering this data over the [network](#networks).
+
+### CID
+A CID (content identifier) is an immutable identifier for a discrete piece of static content stored on [IPFS](#ipfs). CIDs are essentially a hash of the content along with metadata that describes how the content is encoded. Ceramic [streams](#streams) consist of multiple CIDs, encoded using [dag-jose](#dag-jose), and linked together using [IPLD](#ipld).
```

Comment:
Commits are not only encoded using dag-jose, we also use dag-cbor, but other codecs could be used as well.

Reply:
```suggestion
A CID (content identifier) is an immutable identifier for a discrete piece of static content stored on [IPFS](#ipfs). CIDs are essentially a hash of the content along with metadata that describes how the content is encoded. Ceramic [streams](#streams) consist of multiple CIDs, encoded using [dag-jose](#dagjose), and linked together using [IPLD](#ipld).
```